src/foam/u2/view/UnstyledChooseNView.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'UnstyledChooseNView',
+  extends: 'foam.u2.View',
+
+  requires: [
+    'foam.core.SimpleSlot'
+  ],
+
+  properties: [
+    foam.u2.view.ChoiceView.DYNAMIC_SIZE,
+    foam.u2.view.ChoiceView.MAX_SIZE,
+    foam.u2.view.ChoiceView.CHOICES,
+    {
+      name: 'n',
+      class: 'Int'
+    },
+    {
+      name: 'booleanView',
+      class: 'foam.u2.ViewSpec',
+      value: {
+        class: 'foam.u2.CheckBox'
+      }
+    },
+    {
+      name: 'data',
+      class: 'Map',
+      value: {},
+      factory: function () {
+        return {};
+      },
+      adapt: function (_, n) {
```

Comment:
This sets the first 'n' to true/on - for initial display?

Reply:
That's there to allow you to pass an array but... actually right now setting `data` doesn't update the checkboxes so there's no point in adapting this. (originally I had a getter convert the set into an array but the getter conflicted with the `$remove` and `$set` property methods)

I guess my options are either remove this or allow checkboxes to be set based on a provided value for `data`. I think I should do the second thing but that requires some clever thinking